sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/Microsoft.Azure.WebJobs.Extensions.SignalRService.sln

```diff
@@ -6,6 +6,7 @@ MinimumVisualStudioVersion = 15.0.26124.0
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{FFE2667E-131F-40CE-8851-AFAECFBF89BB}"
 	ProjectSection(SolutionItems) = preProject
 		Directory.Build.props = Directory.Build.props
+		..\..\..\eng\Packages.Data.props = ..\..\..\eng\Packages.Data.props
```

Comment:
nit: remove

Reply:
I think it is more convenient to include the file in a solution so that we can update dependencies in a solution.